src/edu/csus/ecs/pc2/ui/SubmitSampleRunsPane.java

```diff
@@ -0,0 +1,877 @@
+		
+		JPanel submitNoSamplesPane = new JPanel();
+		submitNoSamplesPane.setBorder(new TitledBorder(null, "", TitledBorder.LEADING, TitledBorder.TOP, null, null));
+		FlowLayout fl_submitNoSamplesPane = (FlowLayout) submitNoSamplesPane.getLayout();
+		fl_submitNoSamplesPane.setAlignment(FlowLayout.LEFT);
+		submitNoSamplesPane.setBounds(33, 131, 650, 32);
+		controlsPane.add(submitNoSamplesPane);
+
+		checkBoxSubmitFailingSamples = new JCheckBox("Submit Failing Samples");
+		checkBoxSubmitFailingSamples.setVerticalAlignment(SwingConstants.TOP);
+		submitNoSamplesPane.add(checkBoxSubmitFailingSamples);
+		checkBoxSubmitFailingSamples.setSelected(true);
+		checkBoxSubmitFailingSamples.setToolTipText("Submt all non-AC (Yes) submissions");
+
+		judgingTypesButtonGroup.add(submitAllJudgingTypesRadioButton);
+		submitAllJudgingTypesRadioButton.setSelected(true);
+		judgingTypesButtonGroup.add(submitSelectedJudgingTypeRadioButton);
+		
+		submitNoSamplesPane.add(submitAllJudgingTypesRadioButton);
+		submitNoSamplesPane.add(submitSelectedJudgingTypeRadioButton);
+		
+		leftSide.setPreferredSize(new Dimension(230, 230));
+		leftSide.setVerticalScrollBarPolicy(ScrollPaneConstants.VERTICAL_SCROLLBAR_AS_NEEDED);
+		leftSide.setWheelScrollingEnabled(true);
+		splitPane.setLeftComponent(leftSide);
+
+		// centerPane2.add(LogPanel, BorderLayout.SOUTH);
+		JScrollPane logScrollPanel = new JScrollPane(LogPanel);
+		logScrollPanel.setPreferredSize(new Dimension(60, 60));
+		logScrollPanel.setVerticalScrollBarPolicy(ScrollPaneConstants.VERTICAL_SCROLLBAR_AS_NEEDED);
+		logScrollPanel.setWheelScrollingEnabled(true);
+		splitPane.setRightComponent(logScrollPanel);
+		
+		
+		selectedNosLabel.setToolTipText("None selected");
+		selectedNosLabel.addMouseListener(new MouseAdapter() {
+			@Override
+			public void mouseClicked(MouseEvent e) {
+			    showJudgementTypeList();
+			}
+		});
+		submitNoSamplesPane.add(selectedNosLabel);
+		JButton selectJudementTypesButton = new JButton("Select");
+		selectJudementTypesButton.setToolTipText("Select Judgement Types");
+		selectJudementTypesButton.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+			    showJudgementTypeList();
+			}
+		});
+		
+		submitNoSamplesPane.add(selectJudementTypesButton);
+		
+		JLabel lblNewLabel_2 = getWhatsThisOne();
+		submitNoSamplesPane.add(lblNewLabel_2);
+		
+		JPanel submitNoSamplesPane_1 = new JPanel();
+		FlowLayout flowLayout_2 = (FlowLayout) submitNoSamplesPane_1.getLayout();
+		flowLayout_2.setAlignment(FlowLayout.LEFT);
+		submitNoSamplesPane_1.setBorder(new TitledBorder(null, "", TitledBorder.LEADING, TitledBorder.TOP, null, null));
+		submitNoSamplesPane_1.setBounds(33, 161, 650, 32);
+		controlsPane.add(submitNoSamplesPane_1);
+		   
+        checkBoxProblems = new JCheckBox("Problems");
+        checkBoxProblems.setVerticalAlignment(SwingConstants.TOP);
+        checkBoxProblems.setToolTipText("Select Problems");
```

Comment:
Correct me if I'm wrong, but it seems to me that the purpose of the checkbox named `checkBoxProblems` is to indicate whether or not "problems" are to be among the things to be submitted.  Assuming that's correct, it seems like `Select Problems` is a misleading description for this tooltip:  the checkbox isn't about _selecting_ problems but rather about whether the (currently selected) problems are to be _submitted_.  In that case it seems like a better tooltip text would be "Submit Problems" (or maybe "Submit selected problems").